<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1289</width>
    <height>743</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:696;&quot;&gt;Shutter Control&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="ShutterWidget" name="shutter_widget_1" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>554</width>
     <height>238</height>
    </rect>
   </property>
  </widget>
  <widget class="ShutterWidget" name="shutter_widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>80</y>
     <width>554</width>
     <height>238</height>
    </rect>
   </property>
  </widget>
  <widget class="ShutterWidget" name="shutter_widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>554</width>
     <height>238</height>
    </rect>
   </property>
  </widget>
  <widget class="ShutterWidget" name="shutter_widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>370</y>
     <width>554</width>
     <height>238</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ShutterWidget</class>
   <extends>QWidget</extends>
   <header location="global">shutterwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
